kotlin/services/dynamodb/src/main/kotlin/com/kotlin/dynamodb/DynamoDBPartiQ.kt

```diff
@@ -0,0 +1,110 @@
+//snippet-sourcedescription:[DynamoDBPartiQ.kt demonstrates how to work with PartiQL for Amazon DynamoDB.]
+//snippet-keyword:[AWS SDK for Kotlin]
+//snippet-keyword:[Code Sample]
+//snippet-service:[Amazon DynamoDB]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[02/22/2022]
+//snippet-sourceauthor:[scmacdon-aws]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.kotlin.dynamodb
+
+// snippet-start:[dynamodb.kotlin.partiql.import]
+import aws.sdk.kotlin.services.dynamodb.DynamoDbClient
+import aws.sdk.kotlin.services.dynamodb.model.AttributeValue
+import aws.sdk.kotlin.services.dynamodb.model.ExecuteStatementRequest
+import aws.sdk.kotlin.services.dynamodb.model.ExecuteStatementResponse
+// snippet-end:[dynamodb.kotlin.partiql.import]
+
+/**
+To run this Kotlin code example, ensure that you have set up your development environment,
+including your credentials.

```

Comment:
Before running this Kotlin code example, set up your development environment,
including your credentials.